formats/hmrc_manual/publisher/links.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "topics": {
+      "anyOf": [
+        { "$ref": "#/definitions/guid_list" },
+        { "type": "null" }
+      ]
```

Comment:
Yep, @alext, from the looks of existing schemas it I thought so too. But then the schema validation specs in `hmrc-manuals-api` failed (https://github.com/alphagov/hmrc-manuals-api/blob/master/spec/models/publishing_api_manual_spec.rb#L24) with `Errors: ["The property '#/links/topics' of type NilClass did not match the following type: array in schema file:///var/govuk/govuk-content-schemas/dist/formats/hmrc_manual/publisher/schema.json#"]` which I did not understand, but the `"anyOf"` fixed